incubator/astro/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+name: astro
+version: 1.2.0
```

Comment:
Is this chart version 1.2.0?  It would seem that this is chart version 0.0.1 or 1.0.0. 

The chart will almost never stay in-sync with the appVersion due to changes that will need to be made to the chart.  I recommend just versioning the chart like its own project.